its/src/test/java/com/sonar/it/shared/TestUtils.java

```diff
@@ -234,13 +234,11 @@ public static void reset(Orchestrator orchestrator) {
       deleteLocalCache();
     }
 
-    // We add one day to ensure that today's entries are deleted.
-    Instant instant = Instant.now().plus(1, ChronoUnit.DAYS);
-
-    // The expected format is yyyy-MM-dd.
-    String currentDateTime = DateTimeFormatter.ISO_LOCAL_DATE
+    // The expected format is yyyy-MM-ddTHH:mm:ss+HHmm. e.g. 2017-10-19T13:00:00+0200
+    // String currentDateTime = DateTimeFormatter.ISO_OFFSET_DATE_TIME.withZone(ZoneId.of("UTC")).format(now);
+    String currentDateTime = ISO_LOCAL_DATE_TIME
       .withZone(ZoneId.of("UTC"))
-      .format(instant);
+      .format(Instant.now().truncatedTo(ChronoUnit.SECONDS)) + "+0000";
```

Comment:
This solution is like a workaround. We want a specific format, we use something close, trim input, and extend output.
Let's go directly to the point like this:
```
    SimpleDateFormat formatter = new SimpleDateFormat("yyyy-MM-dd'T'HH:mm:ssZ");
    String currentDateTime = formatter.format(new Date());
```
It will produce "current" timezone, but that should not be a problem. As SQ is running on the same machine anyway. If it doesn't work, we can add
```
    formatter.setTimeZone(TimeZone.getTimeZone("UTC"));
```
but I don't see a point in doing it (unless the datetime in Java processing is seriously wrong)

Reply:
The format `yyyy-MM-dd'T'HH:mm:ssZ` does not work. It has to be `017-10-19T13:00:00+0200`

Reply:
` formatter.setTimeZone(TimeZone.getTimeZone("UTC"));` does not work also since it will generate the format with a `Z` at the end that is not accepted by the SQ.